lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -366,33 +366,45 @@ export async function getReleaseNotesMd(
             .replace(regEx(/^\s*#*\s*/), '')
             .split(' ')
             .filter(Boolean);
-          let body = section.replace(regEx(/.*?\n(-{3,}\n)?/), '').trim();
+          const body = section.replace(regEx(/.*?\n(-{3,}\n)?/), '').trim();
```

Comment:
With the changes made here, we are able to make the body variable a `const` - a bit nicer.